src/sql/base/browser/ui/table/plugins/copyKeybind.plugin.ts

```diff
@@ -53,7 +53,6 @@ export class CopyKeybind<T> implements Slick.Plugin<T> {
 		if (handled) {
 			e.preventDefault();
 			e.stopPropagation();
-			e.stopImmediatePropagation();
```

Comment:
Why was this removed?

Reply:
I've reverted the removal, not sure whether they are actually needed, but just in case.